src/Type/Object/Comment.php

```diff
@@ -75,6 +104,17 @@
 		'parent'      => [
 			'type'        => 'Comment',
 			'description' => __( 'Parent comment of current comment. This field is equivalent to the WP_Comment instance matching the WP_Comment->comment_parent ID.', 'wp-graphql' ),
+			'resolve'     => function ( Comment $comment, $args, AppContext $context, ResolveInfo $info ) {
```

Comment:
extra space between function and (

Reply:
Looks like I had a PHPStorm rule set weird for this one. Fixed the rule.